docker-compose.yml

```
@@ -9,14 +9,15 @@ services:
     depends_on:
       - backend-db
     ports:
-      - "8080:8080"
+      - '8080:8080'
     environment:
       - PORT=8080
       - DB_HOST=backend-db
       - DB_PORT=5432
       - DB_USERNAME=postgres
       - DB_NAME=postgres
       - DB_PASSWORD=1234
+      - JWT_KEY=secret
```

```suggestion
      - JWT_KEY=dTH#v1%^K!
```
Думаю, пароль надо установить чуть по сложнее, например, как это.

---

исправлено